exercises/risk-risiko.cpp

```diff
@@ -27,3 +26,52 @@
   M 3 vs 3 => blue win
   O 2 vs 1 => red win
 */
+#include <iostream>
+#include <ctime>
+using namespace std;
+
+void swap(int& a, int& b) {
+    int temp = a;
+    a = b;
+    b = temp;
+}
+
+void selectionsort(int array[], int n) {
+    int indexMax;
+    for (int i = 0; i < n; i++) {
+        indexMax = i;
+        for (int j = i + 1; j < n; j++) {
+            if (array[j] > array[indexMax])
+                indexMax = j;
+        }
+        swap(array[i], array[indexMax]);
+    }
+}
+
+int main() {
+    srand(time(0));
+    int Red[3] = { rand() % 7,rand() % 7 ,rand() % 7 };
+    int Blue[3] = { rand() % 7,rand() % 7 ,rand() % 7 };
+    bool flag;
```

Comment:
replace this flag with a string that will contain the string "Red Wins! \n" or "BLue Wins! \n" and write directly
cout << win_string;
In this way you'll save lines of code

Or, if you want to keep this structure you could remove the flag variable and put instead of if(flag) the if (Red[i] > Blue[i])

in a way or in another the flag variable is useless